SConstruct

```diff
@@ -349,7 +349,8 @@ if selected_platform in platform_list:
         if (env["warnings"] == 'extra'):
             env.Append(CCFLAGS=['-Wall', '-Wextra'] + all_plus_warnings + shadow_local_warning)
         elif (env["warnings"] == 'all'):
-            env.Append(CCFLAGS=['-Wall'] + shadow_local_warning)
+            # FIXME: enable -Wimplicit-fallthrough once #26135 is fixed
+            env.Append(CCFLAGS=['-Wall', '-Wextra', '-Wno-implicit-fallthrough', '-Wno-unused-parameter'] + shadow_local_warning)
```

Comment:
It would make more sense to default to `warnings=extra` on GCC/Clang IMO than to add `-Wextra` in the `all` option.